aleph_export/harvest_aleph_marc.py

```diff
@@ -68,11 +68,12 @@ def process_marc_records_from_stream(self, test_mode_flag: bool = False) -> int:
                 if json_record:
                     aleph_id = json_record.get("id", "")
                     if aleph_id > self.event['maxAlephIdProcessed']:
-                        print("Aleph identifier ", aleph_id, " - ", int(time.time() - self.start_time), " seconds.")
-                        json_record = add_files_to_json_object_class.add_files(json_record)
-                        json_record = standard_json_helpers_class.enhance_standard_json(json_record)
-                        self._save_json_record(json_record)
-                        processed_records_count += 1
+                        if aleph_id in self.event['ids'] or len(self.event['ids']) == 0:
```

Comment:
I don't know but it seems like this filter might cause issues where not all items are processed?

Reply:
It seems to work correctly.  If I have an array of ids, only ids in that array get processed.  If the array is empty, all items get processed.